### PR TITLE
PXC-4199 Release notes for PXC 8.0.32-24

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/release-notes/8.0.31-23.md
new file:   docs/release-notes/8.0.32-24.md
modified:   docs/release-notes/release-notes_index.md
modified:   mkdocs-base.yml
modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0.31-23.md [hunk 1]
```diff
@@ -17,10 +17,9 @@
 
 Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
 
-
 ## Release highlights
 
-Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.0.31-23 (2022-11-21) release notes](https://docs.percona.com/percona-server/8.0/release-notes/8.0.30-22.html).
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.0.31-23 (2022-11-21) release notes](https://docs.percona.com/percona-server/8.0/release-notes/8.0.31-23.html).
```
Comment (patrickbirch) [inline docs/release-notes/8.0.31-23.md:22]: Wouldn't this be 8.0.32?

File: docs/release-notes/8.0.32-24.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* [PXC-4163](https://jira.percona.com/browse/PXC-4163): The `pxc_strict_mode` option did not detect version mismatch.
+
+* [PXC-4170](https://jira.percona.com/browse/PXC-4170): The `galera_restart_on_unknown_option` test failed with assertion.
+
+* [PXC-4176](https://jira.percona.com/browse/PXC-4176): The `galera_sr.mysql-wsrep-features#136` test failed on Jenkins.
+
+## Useful links
+
+The [Percona XtraDB Cluster GitHub location](https://github.com/percona/percona-xtradb-cluster)
+
```
Comment (patrickbirch) [inline docs/release-notes/8.0.32-24.md:65]: Should we add Download here?

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.32-24.md:65]: Added

File: docs/release-notes/8.0.32-24.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+* [PXC-4137](https://jira.percona.com/browse/PXC-4137): The `WSREP` applier threads failed to modify read-only schemas.
+
+* [PXC-4162](https://jira.percona.com/browse/PXC-4162): When doing a rolling upgrade from 5.7 to 8.0, `wsrep_cluster_size` was 0.
+
+* [PXC-4163](https://jira.percona.com/browse/PXC-4163): The `pxc_strict_mode` option did not detect version mismatch.
+
+* [PXC-4170](https://jira.percona.com/browse/PXC-4170): The `galera_restart_on_unknown_option` test failed with assertion.
+
+* [PXC-4176](https://jira.percona.com/browse/PXC-4176): The `galera_sr.mysql-wsrep-features#136` test failed on Jenkins.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.0.32-24.md]: In my opinion, this is "too internal" Would it be possible to mention PXC-4119, PXC-4170, and PXC-4176 as something like "Improvements to MTR tests stability"?

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.32-24.md]: removed the mentioned Jiras

**Comment by alina-derkach-oaza** [comment]
The preview link 
https://test-pxc-docs-8-0.onrender.com/release-notes/8.0.32-24.html

**Review by patrickbirch — APPROVED** [review]

**Pull request merged**